src/main/java/com/google/sps/utils/UserUtils.java

```diff
@@ -206,10 +210,10 @@ public static void addPoints(Entity userEntity, int numPoints, DatastoreService
   public static void addUploadPoints(Entity userEntity, DatastoreService datastore){
 
     QuizTimingPropertiesUtils utils = new QuizTimingPropertiesUtils();
-    if(userEntity.getProperty("lastAwardedUploadPoints") == null || utils.isTimestampOutdated((long) userEntity.getProperty("lastAwardedUploadPoints"))){
+    User user = new User(userEntity);
+    if(user.getLastAwardedUploadPoints() == 0L || utils.isTimestampOutdated(user.getLastAwardedUploadPoints())){
```

Comment:
So for QuizTimePropertiesUtils you shouldn't have to create an instance of it because all the methods are static.

Reply:
Okay cool, that's changed